src/main/java/edu/mbrooks/advancedjava/impl/CollectableImpl.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Program Name:    Collectable.java
+ * Author:          Michael Brooks
+ * Version:         Version 1.0
+ * Course:          Advanced Java Programming, Summer 2018
+ * Copyright:       (C) 2018 Michael A. Brooks
+ * Description:     Implementation of the collectable interface
+ */
+
+package edu.mbrooks.advancedjava.impl;
+
+import edu.mbrooks.advancedjava.interfaces.Icollectable;
+
+/**
+ *
+ * @author michaelbrooks
+ * This is a class to setup if there is a collection of valuables, or a bunch of trash
+ */
+public class CollectableImpl implements Icollectable {
```

Comment:
as you find out as you take this course, I am pretty picky about the names of things. one of things I do not like (even though it is somewhat common) is to use the suffix "impl" on class names. Impl is a detail that should be obvious from the context of the classes' use. It makes the class name less English like and therefore harder to read. 
Basically, you don't need it. You can determine if the class is an interface or a concrete class without this suffix.

Reply:
Understand.